### PR TITLE
STOC-32: Feature/setup api call to retrieve data and return price

### PR DESCRIPTION
**Pull request by Roboto0**

### CODE REVIEW

File: stockAndPrices/services/app/games.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        if not all_ids:
+            all_ids += rs['gameID']
+        else:
+            all_ids += ',' + rs['gameID']
+        count += 1
+        if count == 25:
+            findGamePriceById(all_ids, all_prices)
+            count, all_ids = 0, ''
+
+    if not all_ids and not all_prices:
```
Comment (allenaerostar) [inline stockAndPrices/services/app/games.py:35]: Would we able to determine there are no results before executing the loop on line 25-33?

File: stockAndPrices/services/app/games.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    elif all_ids:
+        findGamePriceById(all_ids, all_prices)
+    return all_prices
+
+def validation(title):
+    if not title:
+        return ""
+    
+    if not title.isalnum():
+        ret = ''
```
Comment (allenaerostar) [inline stockAndPrices/services/app/games.py:46]: What is the intent for creating this string, if its not alphanumeric, shouldn't we return "" or some error?

File: stockAndPrices/services/app/games.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+            findGamePriceById(all_ids, all_prices)
+            count, all_ids = 0, ''
+
+    if not all_ids and not all_prices:
+        return "No Result"
+    elif all_ids:
+        findGamePriceById(all_ids, all_prices)
+    return all_prices
+
+def validation(title):
```
Comment (allenaerostar) [inline stockAndPrices/services/app/games.py:41]: Should rename to validate(title), or a verb

File: stockAndPrices/services/app/games.py (hunk 2 above)
Comment (Roboto0) [reply stockAndPrices/services/app/games.py:46]: if the title included some other symbol such as ":,[]" , I think we want to ignore them, but not the whole string? We only want the alphanumeric character inside the string.

File: stockAndPrices/services/app/games.py (hunk 1 above)
Comment (Roboto0) [reply stockAndPrices/services/app/games.py:35]: Yes, updated. If resp is empty then there is no result

File: stockAndPrices/services/app/games.py (hunk 3 above)
Comment (Roboto0) [reply stockAndPrices/services/app/games.py]: Updated

**Review by lilianahechen — APPROVED** [review]

**Review by allenaerostar — APPROVED** [review]

**Pull request merged**